src/FTPObject.jl

```diff
@@ -1,13 +1,9 @@
-# FTP code for when the file transfer is complete.
-const complete_transfer_code = 226
-
-
 mutable struct FTP
     ctxt::ConnContext
 
     function FTP(ctxt::ConnContext)
         ftp = new(ctxt)
-        @compat finalizer(close, ftp)  # 0.7.0-DEV.2562
+        finalizer(close, ftp)  # 0.7.0-DEV.2562
```

Comment:
Probably don't need the version number in a comment, since it doesn't make sense without the `@compat` annotation.

Reply:
errr I meant to delete that like 3 times, and I kept on forgetting. Thanks for the catch!